packages/client/mutations/NavigateMeetingMutation.ts

```diff
@@ -183,6 +192,38 @@ const NavigateMeetingMutation = (
               setLocalStageAndPhase(store, meetingId, completedStageId)
             }
           }
+          if (phaseType === REFLECT) {
```

Comment:
whoaaaa, double duty!
why not just wait for the sort order to come back from the server?

Reply:
I saw flickering without this part in optimistic updater

Reply:
hmmm, lemme see if i can find a way around that.
my concern is that we're requiring a whole lot more code (the clustering algo, etc), all the reflection entities, and performing a whole lot of extra work & it only benefits the facilitator for a split second.

Reply:
@tianrunhe check out L190 above. for the vote phase, we undo changing the facilitatorStageId. that avoids the flicker!